3rd-party/3rd-party.php

```diff
@@ -10,3 +10,10 @@
 require_once( JETPACK__PLUGIN_DIR . '3rd-party/bitly.php' );
 require_once( JETPACK__PLUGIN_DIR . '3rd-party/bbpress.php' );
 require_once( JETPACK__PLUGIN_DIR . '3rd-party/woocommerce.php' );
+
+if ( class_exists( 'WP_Polldaddy' ) ) {
```

Comment:
What do you think about moving this to a new `3rd-party/polldaddy.php` file to keep things consistent with the existing content?